src/components/AddTodoForm/AddTodoForm.tsx

```diff
@@ -0,0 +1,83 @@
+import { useState } from 'react';
+import users from '../../api/users';
+
+type Props = {
+  addTodo: (title: string, userId: number) => void;
+};
+
+export const AddTodoForm = (props: Props) => {
+  const [title, setTitle] = useState('');
+  const [hasTitleError, setHasTitleError] = useState(false);
+
+  const [userId, setUserId] = useState(0);
```

Comment:
The initial state for `userId` is set to `0`, which is also used as a placeholder value in the select dropdown. Ensure that this does not lead to confusion or errors, especially since `0` is not a valid user ID.